Various issues (#635)
usnistgov/FIPS201: _FIPS201/requirements.md

The PIN on a PIV Card may need to be reset if the cardholder has
forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an
invalid PIN more than the allowed number of retries. A maximum of 10 consecutive PIN retries **SHALL** be permitted unless a lower limit is stipulated by the department or agency.

> Cardholders **MAY** change their PINs at any time by providing the current PIN and the new PIN values as specified in [[SP 800-73]](../_Appendix/references.md#ref-SP-800-73).

Reviewer: ```suggestion
Cardholders **MAY** change their PINs at any time by providing the current PIN and the new PIN values, as specified in [[SP 800-73]](../_Appendix/references.md#ref-SP-800-73).
```

    
    * The cardholder initiates a PIN reset with the issuer operator.
    * The operator authenticates the owner of the PIV Card through an independent

>         procedure such as authenticating the cardholder with an associated derived PIV credential or corfirming reset via email to the on-record goverment issued email address.

Reviewer 2: ```suggestion
        procedure such as authenticating the cardholder with an associated derived PIV credential or confirming reset via email to the on-record government-issued email address.
```

>         procedure such as authenticating the cardholder with an associated derived PIV credential or corfirming reset via email to the on-record goverment issued email address.

Reviewer 2: ```suggestion
        procedure, for example authenticating the cardholder with an associated derived PIV credential or corfirming reset via email to the on-record goverment issued email address.
```

>         procedure such as authenticating the cardholder with an associated derived PIV credential or confirming reset via email to the on-record government-issued email address.

Reviewer 2: ```suggestion
        procedure, for example by authenticating the cardholder with an associated derived PIV credential or by confirming reset via email to the on-record government-issued email address.
```
